config.yaml

```
@@ -34,6 +34,10 @@ steps:
           target: 'LICENSE'
   - type: generateBuildArtifact
   - type: mirrorProcessedFiles
+  - type: runCommand
+    options:
+      command: "composer normalize"
+      skipConfirmation: true
```

Suggest a `composer install` to the user:

```suggestion
  - type: runCommand
    options:
      command: "composer install && composer normalize"
```